ydb/scheme.py

```diff
@@ -23,6 +23,11 @@ class SchemeEntryType(enum.IntEnum):
     SEQUENCE = 15
     REPLICATION = 16
     TOPIC = 17
+    UNKNOWN = 1001
```

Comment:
new code - workaround too, it has universal decisions, which allow grow original codes unlimited and never conflict.

Reply:
fixed it and add some tests